contracts/staking/contracts/src/sys/MixinParams.sol

```diff
@@ -184,4 +281,47 @@ contract MixinParams is
             ));
         }
     }
+
+    /// @dev Asserts that passed in addresses are non-zero.
+    /// @param _wethProxyAddress The address that can transfer WETH for fees.
+    /// @param _ethVaultAddress Address of the EthVault contract.
+    /// @param _rewardVaultAddress Address of the StakingPoolRewardVault contract.
+    /// @param _zrxVaultAddress Address of the ZrxVault contract.
+    function _assertValidAddresses(
```

Comment:
Since these addresses should all be contracts and this function will be called extremely infrequently, I think it might be worth checking that these accounts are in fact deployed contracts. I think this could be good for trustlessness and as a sanity check on the addresses that are registered.

Reply:
Is it really trustless if it's just a contract that allows someone to arbitrarily withdraw funds? 🤷‍♂